docker-compose.yml

```diff
@@ -18,17 +18,22 @@ services:
     image: postgres:latest
     container_name: db-app
     env_file: .env
-    # ports:
-    #   - "5432:5432" # default port for postgres.. no need to expose it here..
     volumes:
       - postgres_data:/var/lib/postgresql/data/
 
   redis:
     image: redis:latest
     container_name: redis-app
-    # ports:
-      # - "6379:6379" # ..it seems that the school runs REDIS on this port...
-      # - "6380:6379"
+
+  # nginx:
+  #   build:
+  #     context: ./nginx
+  #   container_name: nginx-proxy
+  #   ports:
+  #     - "80:80"
+  #     - "443:443"
+  #   depends_on:
+  #     - web
 
```

Comment:
I see some nginx configuration. Is this basically the config from your previous pull request #58 but with the portion in the docker-compose.yml file commented out so we can still develop on the school computers?
Just asking to verify I understand this portion
Thanks!

Reply:
That is correct. The Nginx part is there, just not active.